internal/main_vm.go

```diff
@@ -31,6 +34,27 @@ func Main() {
 	}
 }
 
+// Find the path to package main by looking at the root Caller.
+func findMainPath() string {
+	pc := make([]uintptr, 20)
+	_ = runtime.Callers(0, pc)
+	frames := runtime.CallersFrames(pc)
```

Comment:
And then use the return value from `runtime.Callers` here to trim the slice.

```suggestion
	frames := runtime.CallersFrames(pc[:n])
```